src/workflows/workflows.ts

```diff
@@ -425,4 +429,72 @@ export class Workflows {
       metadataSchemaRequest,
     });
   }
+
+  public async listMetadataRefreshes(
+    ethSigner: EthSigner,
+    collectionAddress?: string,
+    pageSize?: number,
+    cursor?: string,
+  ) {
+    const imxAuthHeaders = await generateIMXAuthorisationHeaders(ethSigner);
+    const ethAddress = await ethSigner.getAddress();
+
+    return this.metadataRefreshesApi.getAListOfMetadataRefreshes({
+      xImxEthSignature: imxAuthHeaders.signature,
+      xImxEthTimestamp: imxAuthHeaders.timestamp,
+      xImxEthAddress: ethAddress,
+      collectionAddress,
+      pageSize,
+      cursor,
+    });
+  }
+
+  public async getMetadataRefreshErrors(
+    ethSigner: EthSigner,
+    refreshId: string,
+    pageSize?: number,
+    cursor?: string,
+  ) {
+    const imxAuthHeaders = await generateIMXAuthorisationHeaders(ethSigner);
+    const ethAddress = await ethSigner.getAddress();
+
+    return this.metadataRefreshesApi.getMetadataRefreshErrors({
+      xImxEthSignature: imxAuthHeaders.signature,
+      xImxEthTimestamp: imxAuthHeaders.timestamp,
+      xImxEthAddress: ethAddress,
+      refreshId,
+      pageSize,
+      cursor,
+    });
+  }
+
+  public async getMetadataRefreshResults(
+    ethSigner: EthSigner,
+    refreshId: string,
+  ) {
+    const imxAuthHeaders = await generateIMXAuthorisationHeaders(ethSigner);
+    const ethAddress = await ethSigner.getAddress();
+
+    return this.metadataRefreshesApi.getMetadataRefreshResults({
+      xImxEthSignature: imxAuthHeaders.signature,
+      xImxEthTimestamp: imxAuthHeaders.timestamp,
+      xImxEthAddress: ethAddress,
+      refreshId,
+    });
+  }
+
+  public async requestMetadataRefresh(
```

Comment:
Again as above, could this please be `createMetadataRefresh`?

Reply:
[6c49010](https://github.com/immutable/imx-core-sdk/pull/175/commits/6c49010930cb947de387c2a65d69188f659f1204)